src/scripts/main.js

```diff
@@ -1,3 +1,306 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+const rows = document.querySelectorAll('tr');
```

Comment:
Consider using `tableBody.querySelectorAll('tr')` instead of `document.querySelectorAll('tr')` to select only rows within the table body.